api-controllers/ServiceBrokerApiController.js

```diff
@@ -41,9 +41,10 @@ class ServiceBrokerApiController extends FabrikBaseController {
 
   getCatalog(req, res) {
     /* jshint unused:false */
-    res.status(CONST.HTTP_STATUS_CODE.OK).json(utils.getPlatformManager({
-      platform: req.params.platform
-    }).getCatalog(catalog));
+    return Promise.try(() => lib.loadServices())
+      .then(() => res.status(CONST.HTTP_STATUS_CODE.OK).json(utils.getPlatformManager({
+        platform: req.params.platform
+      }).getCatalog(catalog)));
```

Comment:
Done the changes.

SFPlans and SFServices controllers are part of interoperator but currently the watch feature is not implemented. Hence the loading of the services are done currently only for the K8s based deployment. When the controllers are added, it can be improved.